.github/workflows/pytest.yml

```diff
@@ -25,8 +25,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pytest
+        python -m pip install pytest
     - name: Test dependencies
       run: |
-        pip install -e .[all]
-        pytest tests
+        python -m pip install -e .[test]
```

Comment:
If you're going to use Flit you can also use https://flit.pypa.io/en/latest/cmdline.html#flit-install to manage this step as well.